### PR TITLE
Wallet Prefab PR 2, Prefabs, Sprites and UI scripts

### PR DESCRIPTION
**Pull request by rob1997**

### CODE REVIEW

**Review by juans-chainsafe — APPROVED** [review]
proper testing will be done when the prefabs are in a sample

File: Packages/io.chainsafe.web3-unity/Runtime/Scripts/UI/LoadingOverlay.cs [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+namespace ChainSafe.Gaming
+{
+    public class LoadingOverlay : MonoBehaviour
+    {
+        [SerializeField] private RectTransform spinnerTransform;
+        [SerializeField] private float spinSpeed = 1f;
+
+        private void Update()
+        {
+            spinnerTransform.rotation *= Quaternion.AngleAxis(spinSpeed, Vector3.forward);
```
Comment (kantagara) [inline Packages/io.chainsafe.web3-unity/Runtime/Scripts/UI/LoadingOverlay.cs:16]: it brings me joy to see at least _some_ 'true' game development code 😁

File: Packages/io.chainsafe.web3-unity/Runtime/Scripts/UI/Resizer.cs [hunk 2]
```diff
@@ -0,0 +1,69 @@
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+namespace ChainSafe.Gaming
+{
+    [RequireComponent(typeof(RectTransform))]
+    public class Resizer : MonoBehaviour
```
Comment (kantagara) [inline Packages/io.chainsafe.web3-unity/Runtime/Scripts/UI/Resizer.cs:9]: What is the exact purpose of this? To make the modal fit all resolutions?

File: Packages/io.chainsafe.web3-unity/Runtime/Sprites/MetaMask_Icon.png
Comment (kantagara) [inline Packages/io.chainsafe.web3-unity/Runtime/Sprites/MetaMask_Icon.png:1]: Is metamask part of our core package for webGL? If not, it shouldn't be part of our core package.

**Review by kantagara — APPROVED** [review]

File: Packages/io.chainsafe.web3-unity/Runtime/Scripts/UI/Resizer.cs (hunk 2 above)
Comment (rob1997) [reply Packages/io.chainsafe.web3-unity/Runtime/Scripts/UI/Resizer.cs:9]: yes, as a percentage of screen height, WebGL modal becomes scrollable when height is too low but stretched with taller screens. Resizes based on screen height percentage.

File: Packages/io.chainsafe.web3-unity/Runtime/Sprites/MetaMask_Icon.png
Comment (rob1997) [reply Packages/io.chainsafe.web3-unity/Runtime/Sprites/MetaMask_Icon.png:1]: it is, I'll create an issue for moving it out

Comment (rob1997) [reply Packages/io.chainsafe.web3-unity/Runtime/Sprites/MetaMask_Icon.png:1]: here it is #1063

**Pull request merged**